dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/model/dataRequest/BulkDataRequest.kt

```diff
@@ -18,9 +18,8 @@ data class BulkDataRequest(
             type = "string",
             example = "[\"DE0005190003\", \"exampleForAnInvalidIdentifier\"]",
         ),
-        schema = Schema(type = "string"),
```

Comment:
Aus Interesse. Why? (Verstehe auch nicht wieso vorher notwendig)

Reply:
Also es war wohl nicht notwendig, hat in meinen open api docs keinen Unterschied mehr gemacht. Ich habe es entfernt, weil es verhindert hat, dass ein array als set ('uniqueIrgendwas') im openApi.json markiert wird und diese Annotation diese property nicht von Haus aus mitliefert.